evaluation-observe/deploy-langfuse-on-ecs-fargate-with-typescript-cdk/cfn_bootstrap.yml

```diff
@@ -0,0 +1,453 @@
+---
+AWSTemplateFormatVersion: "2010-09-09"
+
+Description: |-
+  Quick-start CloudFormation template to deploy the CDK-based infrastructure sample via AWS
+  CodeBuild - with no local CDK/Docker/Python/etc developer setup required.
+
+  This stack sets up an AWS CodeBuild Project to deploy the main solution and triggers the build
+  once (via AWS Lambda) on stack create, plus again whenever the CodeBuildTrigger resource is
+  updated. On deletion, it triggers another CodeBuild job to attempt to destroy the CDK stack.
+  Grants broad permissions to CodeBuild - not recommended for use in production environments.
+
+Parameters:
+  CodeRepo:
+    Type: String
+    Default: https://github.com/aws-samples/amazon-bedrock-samples/tree/main/evaluation-observe/deploy-langfuse-on-ecs-fargate-with-typescript-cdk
```

Comment:
This should just be the repo URL e.g. `https://github.com/aws-samples/amazon-bedrock-samples` - I don't think it will work with the full path because it's not something you can `git clone` right?

Instead, the path would be reflected with a `cd evaluation-observe/deploy-langfuse-on-ecs-fargate-with-typescript-cdk` command just above the `npm install` in the buildspec itself.

(Or if we want to avoid hard-coding it, could consider making it a CloudFormation parameter and passing to CodeBuild through environment variables like we already do with `CodeRepo` and `CodeRepoBranch`?)

Reply:
Fixed in e43e5e73171d4193e1bc90e630b68f1c8d121086